python/cuml/benchmark/algorithms.py

```diff
@@ -246,6 +428,16 @@ def all_algorithms():
             accepts_labels=True,
             accuracy_function=cuml.metrics.accuracy_score,
         ),
+        AlgorithmFIL(
+            xgb if has_xgboost() else None, 
+            sklearn.ensemble.RandomForestClassifier,
+            tl if has_treelite() else None,
+            cuml.ForestInference,
+            shared_args={"max_depth": 10, "n_estimators": 10},
```

Comment:
Can the model be trained with 100 trees?

Reply:
Yes, this can be done by using `--param-sweep n_estimators=[10,100]`, for example